roles/vault/tasks/bootstrap/start_vault_temp.yml

```diff
@@ -13,6 +13,10 @@
            -v /etc/vault:/etc/vault
            {{ vault_image_repo }}:{{ vault_version }} server
 
+#FIXME(mattymo): Crashes on first start with aufs docker storage. See hashicorp/docker-vault#19
```

Comment:
In that vault issue they mention using the `SKIP_SETCAP` env var. is it worth trying that approach?
   `docker run -e SKIP_SETCAP=true ... `